docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -290,20 +357,51 @@ Here's a simple example:
   <input name="name" />
   <input name="email" />
   <textarea name="message" placeholder="What's the issue?" />
+  <button type="submit">Submit</button>
 </form>
 ```
 
 ```javascript
-import { sendFeedback } from "@sentry/react";
+import { captureFeedback } from "@sentry/react";
 
 document
   .getElementById("my-feedback-form")
   .addEventListener("submit", (event) => {
     const formData = new FormData(event.currentTarget);
-    sendFeedback(formData);
+    captureFeedback({
+      name: formData.get('name'),
+      email: formData.get('email'),
+      message: formData.get('message'),
+    }, {
+      attachments: [] // optional
+    });
     event.preventDefault();
   });
 ```
+```typescript {tabTitle: With Image Upload}
+function MyFeedbackForm() {
+  return (
+    <form id="my-feedback-form" onSubmit={event => {
+      const formData = new FormData(event.currentTarget);
+
+      Sentry.captureFeedback({
+        name: formData.get('name'),
+        email: formData.get('email'),
+        message: formData.get('message'),
+      }, {
+        attachments: [], // TODO
+      });
+      event.preventDefault();
+    }}>
+      <input name="name" />
+      <input name="email" />
+      <textarea name="message" placeholder="What's the issue?" />
+      <input type="file" name="attachment" />
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
+```
```

Comment:
@c298lee is going to fixup this snippet so it accepts a file upload and converts it to an Attachment type.